Looping over logs of all jobs using a single command (#18183)
kubernetes/website: content/en/docs/tasks/job/parallel-processing-expansion.md

process-item-cherry-dnfu9   0/1       Completed   0          4m
```

> We can use this single command to check on the output of all jobs at once:

Reviewer: ```suggestion
For this simple example, we can view the output via logs from the pods:
```
(to emphasize that a different approach is appropriate to read output from production jobs, especially if there are very many of them)

Author: @sftim I prefer the original wording. Thank you very much.

```shell

>   kubectl logs -f -l jobgroup=jobexample

Reviewer: ```suggestion
kubectl logs -f -l jobgroup=jobexample
```
